src/main/scala/robosky/ether/block/ZephyrOnionBlock.java

```diff
@@ -0,0 +1,32 @@
+package robosky.ether.block;
```

Comment:
Can we possibly get this converted to scala? Id like to keep the code consistently scala wherever it makes sense to be so (so outside of mixins and structure pieces)